src/Cryptography/Store/InMemoryCipherKeyStore.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Cryptography\Store;
+
+use Patchlevel\EventSourcing\Cryptography\Cipher\CipherKey;
+
+final class InMemoryCipherKeyStore implements CipherKeyStore
```

Comment:
Im not sure if this makes sense to have in our `src` dir. For testing purposes i can understand that, but on production systems?

Reply:
Classes that are used (only) for tests and offered to the developer also belong in the src folder.
I definitely want to test this class too. This is not a stub or mock, but a valid implementation of the interface.